### PR TITLE
1.01

### PR DESCRIPTION
**Pull request by zvezdochiot**
memalign for linux

### CODE REVIEW

**Comment by cpcitor** [comment]
Hello @zvezdochiot . Can you elaborate the rationale for those changes? I had no trouble compiling code on Ubuntu 16.04 without the changes.

File: src/compute.c [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
 #include "ooura/dct.h"
 
+#ifdef _WIN32
```
Comment (cpcitor) [inline src/compute.c]: Why filter FLT_EVAL_METHOD as if it was win32-specific? This is standard C99, ref. https://en.wikipedia.org/wiki/C99#IEEE_754_floating_point_support

**Comment by fidergo-stephane-gourichon** [comment]
Hello @zvezdochiot . Can you elaborate the rationale for those changes? I had no trouble compiling code on Ubuntu 16.04 without the changes.

File: src/compute.c (hunk 1 above)
Comment (fidergo-stephane-gourichon) [inline src/compute.c:15]: Why filter FLT_EVAL_METHOD as if it was win32-specific? This is standard C99, ref. https://en.wikipedia.org/wiki/C99#IEEE_754_floating_point_support

File: src/utils.h [hunk 2]
```diff
@@ -90,7 +93,7 @@ static inline void *alloc_simd(size_t n) {
 #if defined(_WIN32)
         void *p = _aligned_malloc(n, 16);
 #else
-        void *p = aligned_alloc(16, n);
+        void *p = memalign(16, n);
```
Comment (fidergo-stephane-gourichon) [inline src/utils.h:96]: man memalign says `memalign` is obsolete. Are you targeting an old compiling environment?

File: src/compute.c (hunk 1 above)
Comment (zvezdochiot) [reply src/compute.c:15]: compute.c: At top level:
compute.c:15:15: error: expected declaration specifiers or ‘...’ before numeric constant
compute.c:15:37: error: expected declaration specifiers or ‘...’ before string constant

Debian Wheezy

File: src/utils.h (hunk 2 above)
Comment (zvezdochiot) [reply src/utils.h:96]: compute.o:/home/mykaralw/WORK/soft/git/victorvde/jpeg2png/utils.h:93: more undefined references to `aligned_alloc' follow
collect2: error: ld returned 1 exit status

Debian Wheezy.

**Comment by zvezdochiot** [comment]
You can consider me old-fashioned, but malloc.h is checked time.

**Pull request closed without merging**